modules/music_album.rb

```diff
@@ -0,0 +1,18 @@
+# rubocop:disable Metrics/ParameterLists
+require_relative './item'
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(id, genre, author, source, label, published_date,
+                 on_spotify)
+    super(id, genre, author, source, label, published_date)
+    @on_spotify = on_spotify
+  end
+
```

Comment:
Hi Rony, I have one suggestion. The superclass accepts name and publish date as arguments. can you please change the arguments.

![image](https://user-images.githubusercontent.com/116718155/234473262-7ef4fcf8-c678-44eb-9768-2d461e5c17cc.png)